features/events/traps/blueprint-trapd-handler-default.xml

```diff
@@ -0,0 +1,64 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0"
+	xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0 
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+
+	<cm:property-placeholder id="trapHandlerDefaultProperties"
+		persistent-id="org.opennms.netmgt.trapd.handler.default"
+		update-strategy="none">
+		<cm:default-properties>
+			<cm:property name="brokerUri" value="tcp://127.0.0.1:61616" />
+		</cm:default-properties>
+	</cm:property-placeholder>
+
+	<bean id="trapConnectionHandlerDefault"
+		class="org.opennms.netmgt.trapd.TrapNotificationHandlerDefaultImpl" />
+
+	<reference id="trapdConfig" interface="org.opennms.netmgt.config.TrapdConfig" />
+
+	<bean id="trapdConfigProcessor" class="org.opennms.netmgt.trapd.TrapdConfigProcessor">
+		<argument ref="trapdConfig" />
+	</bean>
+
+
+	<bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+		<!-- TODO: HZN-490 Add configurable ActiveMQ URI -->
+		<property name="brokerURL" value="${brokerUri}" />
+		<!-- TODO: HZN-490 Add configurable authentication -->
+		<!-- <property name="userName" value="karaf" /> <property name="password" 
+			value="karaf" /> -->
+	</bean>
+
+	<camelContext id="trapConnectionHandlerDefaultContext"
+		xmlns="http://camel.apache.org/schema/blueprint">
+		<propertyPlaceholder id="properties"
+			location="blueprint:trapHandlerDefaultProperties" />
+
+		<route id="receiveTrapConnection">
+			<from uri="activemq:broadcastTrap" />
+			<convertBodyTo type="org.opennms.netmgt.snmp.TrapNotification" />
+			<!-- <bean ref="unmarshaller"/> -->
+			<!-- Update the TrapdConfig on the message to the local config value -->
+			<bean ref="trapdConfigProcessor" />
```

Comment:
You can comment out the trapdConfigProcessor endpoint, I don't think we need it here.